website/pages/docs/advanced-topics/monitoring.md

```diff
@@ -56,30 +56,21 @@ In production, it is common to use an OpenTelemetry [collector](https://opentele
 
 ### OpenTelemetry and Datadog
 
-In this quick example we will show how to connect an open telemetry collector to Datadog via OpenTelemetry exporter.
+In this example we will show how to send OpenTelemetry traces from the CLI directly to a Datadog agent.
 
-Firstly, you will need to have an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) running either locally or as a gateway. Here is an example of running it locally with docker:
-
-```bash
-docker run  -p 4319:4319 -v $(pwd)/config.yml:/etc/otelcol-contrib/config.yaml otel/opentelemetry-collector-contrib:0.91.0
-```
-
-following is an example for OTEL collector `config.yml` to receive traces locally on 4318 and export them to Datadog:
+First, you will need to [enable OpenTelemetry ingestion on the Datadog Agent](https://docs.datadoghq.com/opentelemetry/interoperability/otlp_ingest_in_the_agent#enabling-otlp-ingestion-on-the-datadog-agent). Add the configuration below to your `datadog.yaml` file:
 
 ```yaml
-receivers:
-  otlp:
+otlp_config:
+  receiver:
     protocols:
       http:
-        endpoint: 0.0.0.0:4318
-exporters:
-  datadog:
-    api:
-      site: "datadoghq.com" # or your tenant site https://docs.datadoghq.com/getting_started/site/
-      key: "<DATADOG_API_KEY>"
+        endpoint: localhost:4318
```

Comment:
You won't be able to send anything to a remote endpoint that's listening only on the localhost address. Since Datadog Agent is likely to run in a Docker container, using the 0.0.0.0 is more likely to work.

```suggestion
        endpoint: 0.0.0.0:4318
```